<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportF30SetupDialog</class>
 <widget class="QDialog" name="ReportF30SetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>параметры отчёта</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="grpUseInputDate">
     <property name="title">
      <string>Использовать дату ввода</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="CDateEdit" name="edtBegInputDate">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblBegInputDate">
        <property name="text">
         <string>Минимальная дата ввода</string>
        </property>
        <property name="buddy">
         <cstring>edtBegInputDate</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblEndInputDate">
        <property name="text">
         <string>Максимальная дата ввода</string>
        </property>
        <property name="buddy">
         <cstring>edtEndInputDate</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="CDateEdit" name="edtEndInputDate">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>80</width>
          <height>19</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="16" column="1" colspan="2">
    <widget class="QFrame" name="frmAge">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="_2">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="edtAgeFrom">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeTo">
        <property name="text">
         <string>по</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="edtAgeTo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>150</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAgeYears">
        <property name="text">
         <string>лет</string>
        </property>
        <property name="buddy">
         <cstring>edtAgeTo</cstring>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>21</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="20" column="0">
    <widget class="QLabel" name="lblTypeFinance">
     <property name="text">
      <string>Тип финансирования</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="lblSex">
     <property name="text">
      <string>По&amp;л</string>
     </property>
     <property name="buddy">
      <cstring>cmbSex</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblEventPurpose">
     <property name="text">
      <string>&amp;Назначение обращения</string>
     </property>
     <property name="buddy">
      <cstring>cmbEventPurpose</cstring>
     </property>
    </widget>
   </item>
   <item row="22" column="0">
    <widget class="QLabel" name="lblVisitPayStatus">
     <property name="text">
      <string>Флаг финансирования</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="lblSocStatusClass">
     <property name="text">
      <string>Класс соц.статуса</string>
     </property>
     <property name="buddy">
      <cstring>cmbSocStatusClass</cstring>
     </property>
    </widget>
   </item>
   <item row="21" column="1" colspan="2">
    <widget class="QComboBox" name="cmbTariff">
     <item>
      <property name="text">
       <string>не учитывать</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>тарифицированные</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>не тарифицированные</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="21" column="0">
    <widget class="QLabel" name="lblTariff">
     <property name="text">
      <string>Тарификация</string>
     </property>
    </widget>
   </item>
   <item row="27" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="lblAge">
     <property name="text">
      <string>Во&amp;зраст с</string>
     </property>
     <property name="buddy">
      <cstring>edtAgeFrom</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="26" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QCheckBox" name="chkVisitHospital">
     <property name="text">
      <string>Учитывать посещения ДС</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QCheckBox" name="chkAmbulator">
     <property name="text">
      <string>Выделить амбулаторные визиты</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="17" column="1" colspan="2">
    <widget class="CSocStatusComboBox" name="cmbSocStatusClass"/>
   </item>
   <item row="12" column="1" colspan="2">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="lblScene">
     <property name="text">
      <string>Место посещения</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbScene"/>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="btnEventTypeList">
     <property name="text">
      <string>Тип обращения</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QLabel" name="lblEventTypeList">
     <property name="text">
      <string>Не задано</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QPushButton" name="btnSpecialityList">
     <property name="text">
      <string>Специальность</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>Подразделение</string>
     </property>
     <property name="buddy">
      <cstring>cmbOrgStructure</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="lblSocStatusType">
     <property name="text">
      <string>Тип соц.статуса</string>
     </property>
     <property name="buddy">
      <cstring>cmbSocStatusType</cstring>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QComboBox" name="cmbSex">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ж</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QCheckBox" name="chkDetailChildren">
     <property name="text">
      <string>Детализация по подросткам</string>
     </property>
    </widget>
   </item>
   <item row="22" column="1" colspan="2">
    <widget class="QComboBox" name="cmbVisitPayStatus">
     <item>
      <property name="text">
       <string>не задано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>не выставлено</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>выставлено</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>отказано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>оплачено</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbEventPurpose"/>
   </item>
   <item row="24" column="0">
    <widget class="QLabel" name="lblIsEventClosed">
     <property name="text">
      <string>Закрытие события</string>
     </property>
    </widget>
   </item>
   <item row="24" column="1" colspan="2">
    <widget class="QComboBox" name="cmbIsEventClosed">
     <item>
      <property name="text">
       <string>не учитывать</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>только завершённые</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>только незавершённые</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="13" column="1" colspan="2">
    <widget class="QLabel" name="lblSpecialityList">
     <property name="text">
      <string>Не задано</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbEventType"/>
   </item>
   <item row="0" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="20" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbTypeFinance"/>
   </item>
   <item row="23" column="1" colspan="2">
    <widget class="QComboBox" name="cmbGrouping">
     <item>
      <property name="text">
       <string>по специальности</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>по должности</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>по профилю оплаты</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>по страховым</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lblEventType">
     <property name="text">
      <string>&amp;Тип обращения</string>
     </property>
     <property name="buddy">
      <cstring>cmbEventType</cstring>
     </property>
    </widget>
   </item>
   <item row="18" column="1" colspan="2">
    <widget class="CRBComboBox" name="cmbSocStatusType"/>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="23" column="0">
    <widget class="QLabel" name="lblGrouping">
     <property name="text">
      <string>Группировка</string>
     </property>
    </widget>
   </item>
   <item row="25" column="0">
    <widget class="QPushButton" name="btnContingentTypeList">
     <property name="text">
      <string>Наблюдаемый контингент</string>
     </property>
    </widget>
   </item>
   <item row="25" column="1" colspan="2">
    <widget class="QLabel" name="lblContingentTypeList">
     <property name="text">
      <string>Не задано</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CSocStatusComboBox</class>
   <extends>QComboBox</extends>
   <header>Registry/SocStatusComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>chkDetailChildren</tabstop>
  <tabstop>chkVisitHospital</tabstop>
  <tabstop>chkAmbulator</tabstop>
  <tabstop>cmbEventPurpose</tabstop>
  <tabstop>btnEventTypeList</tabstop>
  <tabstop>cmbEventType</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>btnSpecialityList</tabstop>
  <tabstop>cmbScene</tabstop>
  <tabstop>cmbSex</tabstop>
  <tabstop>edtAgeFrom</tabstop>
  <tabstop>edtAgeTo</tabstop>
  <tabstop>cmbSocStatusClass</tabstop>
  <tabstop>cmbSocStatusType</tabstop>
  <tabstop>cmbTypeFinance</tabstop>
  <tabstop>cmbTariff</tabstop>
  <tabstop>cmbVisitPayStatus</tabstop>
  <tabstop>cmbGrouping</tabstop>
  <tabstop>cmbIsEventClosed</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportF30SetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportF30SetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
